src/mono/mono/mini/interp/transform-simd.c

```diff
@@ -900,6 +901,9 @@ interp_emit_simd_intrinsics (TransformData *td, MonoMethod *cmethod, MonoMethodS
 	if (image != mono_get_corlib ())
 		return FALSE;
 
+	if (!interp_simd_enabled)
+		return FALSE;
+
```

Comment:
@kg with this I am able to build and run browser-bench sample with and without SIMD enabled. Does it look right or did I disable too much parts here?

Reply:
This looks right to me, and afaik the only place we explicitly use simd opcodes in the interp is in the C intrinsics which will no longer get emitted

Reply:
I think I will need to split the interp-simd.c to all platforms and wasm specific. the build is now broken on other platforms. this way we can probably still get some perf speedup on Vector128, which I think currently doesn't use wasm simd instructions, right?

Reply:
In the end I didn't split it, just fixed the other platforms and wasi builds. We can split it later if needed.